### ISSUE TITLE
Can a computer be used as a node and a client at the same time

### ISSUE THREAD

**Issue by agui009** [ooibc88/blockbench#75]
Can a computer be used as a node and a client at the same time, or is an IP included in the hosts and clients files. Does this affect the test results

**Comment by RUAN0007** [comment]
No. But take a notice that servers and clients must use different ports.
